varats/varats/tools/research_tools/vara.py

```diff
@@ -374,8 +374,58 @@ def verify_install(self, install_location: Path) -> bool:
         status_ok &= (install_location / "bin/opt").exists()
         status_ok &= (install_location / "bin/phasar-llvm").exists()
 
+        # Check that clang++ can display it's version
+
+        clang = local[str(install_location / "bin/clang++")]
+
+        (ret, stdout, _) = clang.run("--version")
+
+        vara_name = self.code_base.get_sub_project("vara-llvm-project").name
+        status_ok &= (ret == 0)
```

Comment:
```suggestion
        status_ok &= (ret == 0)
```
What's the idea behind putting braces around the comparision?